src/main/java/by/exadel/internship/service/impl/UserServiceImpl.java

```diff
@@ -149,4 +150,19 @@ public void updateTimeSlot(UUID userId, List<UserTimeSlotDTO> userTimeSlotDTOLis
         return mapper.map(users);
     }
 
+    @Override
+    public Set<UserDTO> getUsersBySkills(List<String> skills) {
+        List<String> enumToStringList = Arrays.stream(Skill.values())
```

Comment:
Yo don't need custom validation of Enum values. If your endpoint recceive List<Skills> it will throw BadRequest by default if there will be invalid value

Reply:
I returned back previous variant. Yes, I got it